config/csp.config.js

```diff
@@ -1,16 +1,17 @@
 // docs: https://helmetjs.github.io/docs/csp/
 
-const scriptSrc = ["'self'", "'unsafe-inline'", 'cdnjs.cloudflare.com']
+const scriptSrc = ["'self'", "'unsafe-inline'", 'cdnjs.cloudflare.com', '*.herokuapp.com']
 
 if (process.env.NODE_ENV === 'development') {
   scriptSrc.push("'unsafe-eval'")
 }
 
 module.exports = {
   defaultSrc: ["'self'"],
-  scriptSrc,
+  scriptSrc: scriptSrc,
```

Comment:
I guess I didn't need to do this, but I think the normal `key: value` syntax is less confusing.